src/Microsoft.Identity.Client/AppConfig/MatsConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class MatsConfig : IMatsConfig
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public string AppName { get; set; }
```

Comment:
Can this object be made immutable?

Reply:
It can't because the intention is that you could load this from a config file and those types have to be get/set for the config loading to work.

---
In reply to: [270951592](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/955#discussion_r270951592) [](ancestors = 270951592)